frontend/src/pages/org/workflow-editor.ts

```diff
@@ -487,6 +489,13 @@ export class CrawlConfigEditor extends LiteElement {
       return fallback;
     };
 
+    const bytesToGB = (value: any, fallback: number | null) => {
+      const bytesPerGB = 1073741824;
```

Comment:
Could store this constant outside of the class definition, around line 216 like `DEFAULT_BEHAVIORS`, to use on line 2139

Reply:
Updated